src/Features/Core/Portable/StackTraceExplorer/ParsedFrame.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.StackTraceExplorer
+{
+    internal class ParsedFrame
```

Comment:
this shoudl be abstract.  if concrete, it should be sealed.  we should not have concrete-deriving-from-concrete inheritance.